cmd/bisync/log.go

```diff
@@ -29,16 +29,29 @@ func (b *bisyncRun) indent(tag, file, msg string) {
 		logf = fs.Logf
 	}
 
+	if b.opt.DryRun {
+		logf = fs.Logf
+		msg += " in dry-run mode"
```

Comment:
The `logf` part is great! I'm not sure we need the `msg` part. I think it's already clear we're in dry run mode from other lines, like `file.txt: Skipped copy as --dry-run is set`

It would also read a little awkwardly in some places, like:

```
NOTICE: - Path2    Do queued copies to in dry-run mode - Path1
```

Reply:
I'll remove the msg